zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1116,6 +1116,13 @@ As an example, this will enable all four letter word commands:
     so as to prevent unexpected additional load on the voting peers during
     the process. Defaults to 200 ms.
 
+* *localSessionsEnabled* and *localSessionsUpgradingEnabled* :
+   **New in 3.5:**
+   Optional value is true or false. their default value are false,
+   Turning on the local session feature by setting *localSessionsEnabled=true*.Turning on
+   *localSessionsUpgradingEnabled* can make the local session upgrade to
+   the global session automatically, which only matters when *localSessionsEnabled* is enabled.
```

Comment:
... can upgrade a local session to a global session automatically as required(e.g. creating ephemeral nodes)...